CONTRIBUTING.md

```diff
@@ -108,6 +108,8 @@ Follow the instructions below on how to install Bundler and setup the database.
 * Get set up: `./script/setup`
 * Run the database rake tasks if needed:
     `bundle exec rake db:reset db:test:prepare --trace`
+* Set up elasticsearch indexes:
+    `be rake environment elasticsearch:import:all DIR=app/models FORCE=y`
```

Comment:
It's good to expand `be` to `bundle exec`.  (I know many developers alias `bundle exec` with `be` or `bx`.)

Reply:
lol whoops, sorry.